community-spotlight-personalization-service.mdx

```diff
@@ -0,0 +1,57 @@
+
+- The frontend is all [Next.js](https://nextjs.org/) and hosted on [Vercel](https://vercel.com/)
+- The data pipelines are built with [Inngest](https://www.inngest.com/)
+- [GPT4](https://openai.com/index/gpt-4/) and [Stable Diffusion](https://stability.ai/) are used to generate images based on users prompts
+- The images, JSON, scoring and additional metadata are stored in Xata
+
+On the tech side, it was important to have a service that could allow them to consolidate the services used and simplify the architecture. This was why Xata was chosen for their data layer, it checked all the boxes for relational data, search and file storage.
+
+The user experience in the platform is pretty simple. Moodly will ask you 3-4 questions once a month and will generate new images for your devices that take the answers of those questions, and output images meant to improve you mood. Users get a number of credits each month to user for image generation. Dr. Dubz and CJ are just getting started, but the traction they’ve seen so far amongst their community seems promising.
+
+> From a developer perspective, I really love Xata. I’m used to PostgreSQL and felt at home with that setup, but Xata makes Postgres easy. The Airtable-like interface gave us speed, making it easy to interface with the data as a team.\
+> CJ - Moodly developer
+
+## Feedback and feature requests
+
+As the conversation steered towards what’s next, we asked the Moodly team what’s missing from Xata or what else they’d like to see out the platform. Here are some of their favorite parts:
+
+- **Speed of development.** The combination of PostgreSQL, Xata TypeScript SDK and consolidation of services made for quick, iterative development.
+- **Branching.** CJ ended up creating many environments during the initial development of Moodly and post-production.
+- **Collaboration.** Being able to collaborate on data entry, manipulation and management in the spreadsheet-like table made it really easy to move fast.
+
+After asing what they’d like to see from Xata, we talked a bit about roadmap. Here’s what Dr. Dubz and CJ shared.
+
+- **Chat with your data.** As your dataset grows, it gets a bit more complicated to interact with it and get meaningful answers. Being able to chat with your database to explore further was a feature they’d like to see in Xata.
+- **AI pipeline columns.** Today, Moodly uses two separate services to generate images. They’d love for this to be baked into Xata so they can remove complexity from their architecture. Being able to define a column with a model and prompts to have it produce output for a record would fit their use case well.
+- **Protected branches.** There were a few occasions where CJ wish he had a Dr. Dubz-proof production branch. Being able to mark a branch as protected so that the production application is not impacted would have helped in these scenarios.
```

Comment:
```suggestion
- **Protected branches.** There were a few occasions where CJ wished he had a Dr. Dubz-proof production branch. Being able to mark a branch as protected so that the production application is not impacted would have helped in these scenarios.
```